NEWS.md

```diff
@@ -16,6 +16,10 @@ This release is in development. For a stable release install 1.3.5 from CRAN.
   instead of `create_gp_data()`. By @jamesmbaazam in #388 and reviewed by @seabbs.
 * Improved compilation times by reducing the number of distinct stan models and deprecated `tune_inv_gamma()`. By @sbfnk in #394 and reviewed by @seabbs.
 * Changed touchstone settings so that benchmarks are only performed if the stan model is changed. By @sbfnk in #400 and reviewed by @seabbs.
+* `dist_fit()`'s `samples` argument now takes a default value of 1000 instead
```

Comment:
this needs to be over a single line vs having linebreaks. I corrected this in the previous PRs as only just noticed

Reply:
I've fixed this here 07e137b007f40f632f9e7cbbaa21f7e9c9e4f9a0.